opencti-platform/opencti-graphql/src/schema/relationRef-definition.ts

```diff
@@ -8,7 +8,8 @@ export interface RelationRefDefinition {
   stixName: string
   mandatoryType: MandatoryType
   multiple: boolean
-  checker?: Checker // TODO: after migration checker will be mandatory
+  checker?: Checker
```

Comment:
// TODO: after migration checker will be mandatory

Reply:
Have builder instead of static declaration